.gitmodules

```
@@ -0,0 +1,4 @@
+[submodule "ci"]
+	path = ci
+	url = https://github.com/UKHomeOffice/hocs-ci-infrastructure.git
+	branch = main
```

Submodules are always a pain to keep up to date. It might be easier to clone in a pipeline step?


```suggestion
[submodule "ci"]
    path = ci
    url = https://github.com/UKHomeOffice/hocs-ci-infrastructure.git
    branch = main
```